src/Microsoft.IO.RecyclableMemoryStream.csproj

```diff
@@ -10,4 +10,13 @@
     <DelaySign>true</DelaySign>
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard1.4|AnyCPU'">
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Microsoft.IO.RecyclableMemoryStream.xml</DocumentationFile>
```

Comment:
Why not just add this property without conditions--the value is always the same

Reply:
Agreed. I'll try and get this done in the next couple of days.